.golangci.yml

```diff
@@ -3,7 +3,9 @@ run:
   timeout: 10m
   sort-results: true
   allow-parallel-runners: true
-  exclude-dir: testutil/testdata
```

Comment:
can you undo changes to this file please

Reply:
ahhh yes absolutely